src/systems/physics/Physics.cc

```diff
@@ -982,6 +990,7 @@ void PhysicsPrivate::CreateWorldEntities(const EntityComponentManager &_ecm,
         auto worldPtrPhys = this->engine->ConstructWorld(world);
         this->entityWorldMap.AddEntity(_entity, worldPtrPhys);
 
+
```

Comment:
remove

Reply:
Done in https://github.com/gazebosim/gz-sim/pull/1949/commits/af117e398bee602f002317d1ae9808a5d1f7add5